test/grpcecho/echo.proto

```diff
@@ -16,14 +16,14 @@ syntax = "proto3";
 
 package grpecho;
 
-service EchoTestService {
-  rpc Echo(EchoRequest) returns (EchoResponse);
+service GrpcEchoTestService {
```

Comment:
@rshriram why is this changed ?
is that what causes
https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_pilot/1102/pilot-presubmit/574/

Reply:
filled #1112 as I don't understand how it seems to have passed in this pr and not passing anymore

Reply:
@rshriram ping - why the change ?